modules/util/url.go

```diff
@@ -39,3 +39,12 @@ func URLJoin(base string, elems ...string) string {
 	}
 	return joinedURL
 }
+
+func SanitizeURL(s string) (string, error) {
+	u, err := url.Parse(s)
```

Comment:
Make sure url is `urlDecoded`.

Reply:
What do you mean? (For this PR) the url was already parsed somewhere else already.